src/logger.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include "databasetasks.h"
+
+#include <condition_variable>
+#include <fstream>
+#include <mutex>
+#include <queue>
+#include <source_location>
+#include <string>
+#include <thread>
+
+enum LogSeverity
+{
+	LDEBUG = 0,
+	LINFO = 1,
+	LWARNING = 2,
+	LERROR = 3,
```

Comment:
L prefix is added to shut up VS, it's complaining about ERROR(is treated as maro to something)